applications/asset_tracker/README.rst

```diff
@@ -155,6 +155,12 @@ For example, configure ``CONFIG_POWER_OPTIMIZATION_ENABLE`` to enable power opti
 In |SES|, select **Project** > **Configure nRF Connect SDK project** to browse and configure these options.
 Alternatively, use the command line tool ``menuconfig`` or configure the options directly in ``prj.conf``.
 
+Secure Boot
```

Comment:
Leave out the section - it does not adhere to our template. Just add the information a new paragraph.
(We're working on updating the template to add a configuration section, and will move the content into that section then.)

Reply:
But the asset tracker doc has sections with:
- User Interface
- Power Optimization

isn't the description of how to enable the immutable bootloader similar to configuring Power Optimization ?

Reply:
User interface is a section from the template, and Power Optimization shouldn't be where it currently is - it should be a subsection to Overview, and in the future it'll go into the Configuration section as well.

Reply:
> be where it currently is - it should be a subsection to Overview, and in the futu

Thanks for the clarification.